docs/source/server-process.rst

```diff
@@ -36,7 +36,8 @@ pairs.
    * A callable that takes any :ref:`callable arguments <server-process/callable-arguments>`,
      and returns a list of strings that are used & treated same as above.
   
-   This key is required.
+   If the command is not specified or is an empty list, it is assumed by the
+   proxy that the process is already running and available on ``{port}``.
```

Comment:
I'm updating the language here with two goals:

1. "by the proxy" is a too vague reference to jupyter-server-proxy and better omitted
2. referencing that it is available on `port` could be nice, but being that explicit makes this PR conflict with #337 so I'm omitting it to make this documentation not need to be updated if that gets merged.

```suggestion
   If the command is not specified or is an empty list, the server process is
   assumed to be started ahead of time and already available to be proxied to.
```